src/Query/Filter.php

```diff
@@ -36,22 +47,44 @@ public function getLimit(): int
         return $this->limit;
     }
 
+    public function getSortColumn(): ?string
+    {
+        return $this->sortColumn;
+    }
+
+    public function getSortOrder(): string
+    {
+        return $this->sortOrder;
+    }
+
+    public function hasSort(): bool
+    {
+        return $this->sortColumn !== null;
+    }
+
     /**
      * @return array<string,string>
      */
     public function getQueryStringParams(): array
     {
-        return [
+        $return = [
             'offset' => (string) $this->getOffset(),
             'limit' => (string) $this->getLimit(),
         ];
+
+        if ($this->hasSort()) {
+            $return['sort'] = $this->sortColumn.':'.$this->sortOrder;
+        }
+
+        return $return;
     }
 
-    public static function fromRequest(Request $request): self
+    public static function fromRequest(Request $request, ?string $defaultSortColumn = null): self
```

Comment:
just thinking, maybe we can put `name` here as default?

Reply:
At the moment only the Organisation Package list supports sorting, but other tables could. I think it's clearer if the Controller specified the default, to avoid any confusion with the database layer getting a default that it doesn't actually handle

Reply:
:+1: solid argument